server/src/main/java/org/elasticsearch/usage/UsageService.java

```diff
@@ -26,11 +26,13 @@ public class UsageService {
     private final Map<String, BaseRestHandler> handlers;
     private final SearchUsageHolder searchUsageHolder;
     private final CCSUsageTelemetry ccsUsageHolder;
+    private final CCSUsageTelemetry esqlUsageHolder;
 
     public UsageService() {
         this.handlers = new HashMap<>();
         this.searchUsageHolder = new SearchUsageHolder();
         this.ccsUsageHolder = new CCSUsageTelemetry();
+        this.esqlUsageHolder = new CCSUsageTelemetry(false);
```

Comment:
Question: I couldn't find any case where `true` is passed to the constructor. Do we anticipate any such scenarios in the near future?

Reply:
true is the default, so there's no reason to explicitly pass true.